ur20_moveit_config/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.3)
```

```suggestion
cmake_minimum_required(VERSION 3.0.2)
```

I know, the UR16e also still has 2.8.3, this should be fixed, as well. But not as part of this PR, obviously.